lib/features/thread/data/repository/thread_repository_impl.dart

```diff
@@ -63,10 +64,8 @@ class ThreadRepositoryImpl extends ThreadRepository {
         accountId,
         limit: limit,
         sort: sort,
-        filter: emailFilter?.filter,
+        filter: EmailFilterCondition(inMailbox: emailFilter?.mailboxId),
```

Comment:
If only get filter by mailbox. Will other filter conditions be dropped? Let's test the cases to be more sure.

Reply:
other filter conditions had move to filter cache email